wekesa931/Questioner
# [finishes #163080119] creates the login endpoint

#### What does this PR do?
It creates the log-in api endpoint
#### Description of Task to be completed?
To create the log-in endpoint path:
```
GET         /v1/user/auth/login/<int:user_id>

```
This endpoint redirects the user to their account

#### How should this be manually tested?
clone the repository to your local machine
```
$ git clone https://github.com/wekesa931/Questioner.git
```
Set up the virtual environment
```
$ pip install virtualenv
$ virtualenv venv
$ venv source bin/activate
```
run the application
```
$ python run.py
```
Test the endpoint in postman as follows:
1. In postman, go to path
```
 GET         /v1/user/auth/login/<int:user_id>
```
Note that only the users who have signed up will be displayed

#### What are the relevant pivotal tracker stories?
    #163080119
![login](https://user-images.githubusercontent.com/31995387/50884519-2e3dd000-13fc-11e9-9e5b-5fe77e4b4846.png)
